recipes/entrypoints/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp entrypoints.py $SP_DIR
```

Comment:
Thanks for clarifying @takluyver.

Neat I like this idea of `flit`. Not sure what `flonda` does though.

The question of integration with `conda`/`conda-build` is a good one. However, copying over does seem straightforward now that we understand what is happening and does seem fine to me for now.